libs/cms/src/lib/models/sidebarCard.model.ts

```diff
@@ -24,11 +25,15 @@ export class SidebarCard {
   image?: Image | null
 }
 
-export const mapSidebarCard = ({ sys, fields }: ISidebarCard): SidebarCard => ({
+export const mapSidebarCard = ({
+  sys,
+  fields,
+}: ISidebarCard): SystemMetadata<SidebarCard> => ({
+  typename: 'SidebarCard',
   id: sys.id,
   type: fields.type ?? '',
   title: fields.title ?? '',
-  content: fields.content ?? '',
+  contentString: fields.content ?? '',
```

Comment:
Any reason for this?

Reply:
Yes, we can't have two fields with the same name in fragments.ts sadly so I had to rename this